src/chrome/content/rules/Panasonic.com.xml

```diff
@@ -1,38 +1,28 @@
 <!--
-	CDN buckets:
+	Invalid certificate:
+		business.panasonic.com
+		metrics.panasonic.com
+		www-images.panasonic.com
 
-		- wpc.377F.edgecastcdn.net/??377F/
-
-			- news.panasonic.net
-
-
-	Nonfunctional domains:
-
-		- (www.)panasonic.net	(dropped)
-		- news.panasonic.net	(404; mismatched, CN: edgecastcdn.net)
-
-
-	Problematic domains:
-
-		- (www.)panasonic.com		($ redirects to www.panasonic.net)
-		- www-images.panasonic.com	(akamai)
+	Mixed content:
+		news.panasonic.com
 
 -->
 <ruleset name="Panasonic.com (partial)">
 
-	<target host="*.panasonic.com" />
-
-
-	<securecookie host=".*\.panasonic\.com$" name=".+" />
+	<target host="panasonic.com" />
+	<target host="www.panasonic.com" />
+	<target host="metrics.panasonic.com" />
+	<target host="security.panasonic.com" />
+	<target host="shop.panasonic.com" />
+	<target host="www.security.us.panasonic.com" />
```

Comment:
```
aircon.panasonic.com
eu.automotive.panasonic.com
cycle.panasonic.com
estr.panasonic.com
training.eu.panasonic.com
eng.faq.panasonic.com
ftp.panasonic.com
gccatapult.panasonic.com
industrial.panasonic.com
info.panasonic.com
jcms.panasonic.com
ps-hp.jpn.panasonic.com
mobile.panasonic.com
airconditioner.my.panasonic.com
cooking.my.panasonic.com
fan.my.panasonic.com
lifestyle.my.panasonic.com
na.industrial.panasonic.com
na.panasonic.com
pewg.panasonic.com
www.security.us.panasonic.com
jp.store.panasonic.com
av.jpn.support.panasonic.com
viko.panasonic.com
```

Reply:
Not sure what happened to `mobile`, but `https://ftp.panasonic.com/login/` still WFM.